### PR TITLE
Test & Fix for Issue 531

### PR DESCRIPTION
**Pull request by jogibear9988**

### CODE REVIEW

**Comment by jogibear9988** [comment]
Is this fix ok so?

**Comment by jogibear9988** [comment]
@ili is this fix ok so?

**Comment by jogibear9988** [comment]
maybe Guid should also be added as supported type.

File: Source/Linq/Builder/ExpressionBuilder.SqlBuilder.cs [hunk 1]
```diff
@@ -1889,7 +1896,18 @@ ISqlExpression GetParameter(Expression ex, MemberInfo member)
 				accessorExpression = enumMapExpr.GetBody(accessorExpression);
 			}
 
-			var expr = dataContext.MappingSchema.GetConvertExpression(type, typeof(DataParameter), createDefault: false);
+			LambdaExpression expr = null;
+			if (buildParameterType != BuildParameterType.InPredicate
```
Comment (ili) [inline Source/Linq/Builder/ExpressionBuilder.SqlBuilder.cs]: @jogibear9988 may be we should always ignore `ConvertExpression` when building `In`?  I mean if we do have something like `IEnumerable<MyString>` it should be converted only when we are iterating through array?

Comment (jogibear9988) [inline Source/Linq/Builder/ExpressionBuilder.SqlBuilder.cs]: Your 're right, I will change

**Pull request merged**